city_scrapers/spiders/il_investment_policy.py

```diff
@@ -0,0 +1,157 @@
+import re
+from collections import defaultdict
+from datetime import date, datetime, time
+from enum import Enum
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlInvestmentPolicySpider(CityScrapersSpider):
+    name = "il_investment_policy"
+    agency = "Illinois Investment Policy Board"
+    timezone = "America/Chicago"
+    allowed_domains = ["www2.illinois.gov"]
+    start_urls = ["https://www2.illinois.gov/sites/iipb/Pages/MeetingInformation.aspx"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+        links = self._parse_links(response)
+
+        for item in response.xpath(
+            "//h2[text()='Agendas']/following-sibling::ul"
+            "[not(preceding-sibling::h2[text()='Minutes'])]"
+            "/li/p"
+        ):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location={
+                    "name": "James R. Thompson Center",
+                    "address": "100 W. Randolph St., Room 16-503, Chicago, Illinois"
+                },
+                source=response.url
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            day = date(meeting['start'].year, meeting['start'].month, meeting['start'].day)
```

Comment:
It might be quicker to just call `meeting['start'].date()` in `links.get` unless I'm missing something

Reply:
That ends up including the start time (1:30pm), but the links map doesn't care for / include the time. I guess you could get the 'start' date and then zero out the hours and minutes to make it match what the links map expects.

Reply:
`meeting["start"]` should be a `datetime` object, so `date()` should return a `datetime.date` without the time included that should fit the links map. It should essentially be a shortcut for what you're doing here

Reply:
Ah, true! Thanks!